File: Minimum Perimeter of a Rectangle.js
function minimumPerimeter(area) {
  let min = Infinity;
  for (let i = 1; i * i <= area; i++) {
    if (area % i === 0) {
      const h = i;
      const w = area / i;
      const perimeter = 2 * (h + w);
      if (perimeter < min) {
        min = perimeter;
      }
    }
  }
  return min;
}

File: Minimum Perimeter of a Rectangle/Minimum Perimeter of a Rectangle.swift
func minimumPerimeter(_ area: Int64) -> Int64 {
    var minPerimeter = Int64.max
    let limit = Int64(Double(area).squareRoot())
    
    for i in 1...limit {
        if area % i == 0 {
            let h = i
            let w = area / i
            let perimeter = 2 * (h + w)
            minPerimeter = min(minPerimeter, perimeter)
        }
    }
    
    return minPerimeter
}

